app/views/open_access_publications/edit.html.erb

```diff
@@ -106,6 +106,7 @@
                         as: :boolean,
                         required: true,
                         label: "I have read and agree to #{link_to 'the deposit agreement', 'https://scholarsphere.psu.edu/agreement', target: '_blank'}.".html_safe %>
+            <%= f.hidden_field :deputy_user_id, value: current_user.deputy.id %>
```

Comment:
How come we pass the deputy ID through the web form here? Can't we retrieve and save this value entirely on the backend? We don't need to worry much about it being manipulated in this case, but the fact that it _can_ be manipulated smells to me. I don't think that this should be input from the client, even if it's hidden from the user in the UI.

Reply:
I supposed we can just merge the id in the params object in the controller?

Reply:
Yeah, I think that's a much better way to go.